core/src/main/java/nu/yona/server/subscriptions/service/BuddyService.java

```diff
@@ -279,6 +285,7 @@ void removeBuddyInfoForRemovedUser(User user, Buddy buddy)
 		{
 			// Buddy request was accepted
 			removeMessagesSentByBuddy(user, buddy);
+			removeMessagesSentByUserToBuddy(user, buddy);
```

Comment:
For my understanding, is this only required if the user was removed with `UserService.deleteExistingUserToOverwriteIt`? I see that the user at removal at `UserService.deleteUser` already removes the named and anonymous messages in `removeBuddyInfoForBuddy`.